excelutils/excel_sbol_utils/library3.py

```diff
@@ -77,10 +77,13 @@ def subcomponents(rowobj): #UPDATE TO WORK WITH CELL DICT, ALLOW CONSTRAINTS
 		for b in b_split:
 			try:
 				obj = rowobj.obj_dict[b]['object']
-				# rowobj.obj.variable_features.append(obj)
+				temp_var = sbol3.SubComponent(instance_of=obj)
```

Comment:
why are you creating a new object if the object exists?

Reply:
I think line 112 and line 64 are clashing in the case of backbones

Reply:
which makes sense as in line 64 you want a template with no _ins_ in it. Because oldobj needs a template with 2 subcompoents: backbone and insert. The insert subcomponent points to the newobj and the backbone subcomponent points to backbone(s)

Reply:
Ok, thanks.